Sources/AST/ASTPass/ASTPass.swift

```diff
@@ -17,19 +17,23 @@ public protocol ASTPass {
   func process(topLevelModule: TopLevelModule, passContext: ASTPassContext) -> ASTPassResult<TopLevelModule>
 
   // MARK: Top Level Declarations
-  func process(topLevelDeclaration: TopLevelDeclaration, passContext: ASTPassContext) -> ASTPassResult<TopLevelDeclaration>
-  func process(contractDeclaration: ContractDeclaration, passContext: ASTPassContext) -> ASTPassResult<ContractDeclaration>
+  func process(topLevelDeclaration: TopLevelDeclaration,
+               passContext: ASTPassContext) -> ASTPassResult<TopLevelDeclaration>
+  func process(contractDeclaration: ContractDeclaration,
+               passContext: ASTPassContext) -> ASTPassResult<ContractDeclaration>
   func process(structDeclaration: StructDeclaration, passContext: ASTPassContext) -> ASTPassResult<StructDeclaration>
```

Comment:
I understand wanting to break lines to satisfy the linter rules but this leaves the process / postProcess functions inconsistent in their layout.
 
What are your thoughts on breaking all of these functions at passContext or making the first parameter label optional to reduce to line length.

Reply:
Making parameters optional simply to reduce line length seems a bit extreme, especially since this affects the signature of the function. Why do these need to be consistent necessarily? They are readable either way.

Perhaps we should consider using some kind of preprocessing script / macro to generate these ahead of time?

Reply:
It was partly due to this being a design decision we were thinking of previously. 

That does sound sensible to have a script ahead of time. 

I'm still happy to approve the PR without just wanted to start a discussion.